x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -44335,7 +44335,7 @@
     "xpack.spaces.navControl.spacesMenu.changeCurrentSpaceTitle": "更改当前空间",
     "xpack.spaces.navControl.spacesMenu.findSpacePlaceholder": "查找工作区",
     "xpack.spaces.navControl.spacesMenu.noSpacesFoundTitle": "未找到工作区",
-    "xpack.spaces.navControl.spacesMenu.selectSpacesTitle": "您的工作区",
```

Comment:
@ek-so @elena-shostak unless our guidance has changed, we should not be editing translation files directly. Translations are handled by an external provider.

Reply:
@legrego thanks, and I'm sorry. Not sure what should I do about it now?

Reply:
@elastic/kibana-core what is our typical guidance here? Should this PR be reverted and re-applied without translations?